MBINCompiler/Models/NMSTemplate.cs

```
@@ -290,7 +302,17 @@ public XmlElement AppendToXml(XmlElement parentElement, XmlDocument document)
                             ((NMSTemplate)template).AppendToXml(prop3, document);
 
                         break;
-
+                    case "Object":
+                        if (field.DeclaringType.BaseType.Name == "NMSTemplate")
+                        {
+                          try
+                          {
+                            NMSTemplate template = (NMSTemplate)field.GetValue(this);
+                            var element = template.AppendToXml(el, document);
+                            element.SetAttribute("name", fieldName);
+                          } catch {}

```

Is there any reason for swallowing every single exception here?